pkg/migrations/paths.go

```diff
@@ -78,11 +78,20 @@ func GetRestoreClone() string {
 // If path is a symbolic link, remove it and the database it points to.
 func SafeRemoveDBWithSymbolicLink(path string) error {
 	currentDB, err := fileutils.ResolveIfSymlink(CurrentPath())
-	utils.CrashOnError(errors.Wrap(err, "no current database"))
+	if err != nil {
+		if features.PostgresDatastore.Enabled() {
+			log.Infof("in Postgres mode with no current RocksDB database: %v", err)
+		} else {
+			utils.CrashOnError(errors.Wrap(err, "no current database"))
+		}
+	}
 
 	switch path {
 	case CurrentPath(), currentDB:
-		utils.CrashOnError(errors.Errorf("Database in use. Cannot remove %s", path))
+		if !features.PostgresDatastore.Enabled() {
```

Comment:
This function is to protect the current database from being accidentally removed from auto or manual linkage errors. Let's not change this. You can just use the dedicated function DecommissionRocksDB to remove them.